frontend/src/features/entity_selector/EntitySelectButton.tsx

```diff
@@ -157,6 +158,14 @@ const VideoInput = ({
         },
         disabled: !isLoggedIn || !otherUid,
       },
+      {
+        name: 'good-short-videos',
+        label: t('entitySelector.goodShortVideos'),
+        fetch: async () => {
+          return await getGoodShortVideos();
+        },
+        disabled: !isLoggedIn,
+      },
```

Comment:
(a) Done in 37745a2c85fdbd42294c3ab4ef66637586b4c32b. The dialog still required an increased width to avoid the horizontal scrolling on the tab names.

(b) Done in 42d71c2b96b9d3dcddd696b4f94507569cc358aa